assignment4.Rmd

```diff
@@ -115,4 +123,21 @@ biplot(pca)
 # Part III  
 
 ## Also in this repository is a data set collected from class that shows how many students thought that a TC program was related to andother TC program. Use PCA to look for components of related programs. Explain why you think there are relationships between these programs.
+```{r}
+dt1 <- read.csv("TCprograms.csv",sep = "")
+rownames(dt1) <- dt1$program
+dt1<- dt1[,-1]
+pca3 <- prcomp(dt1, scale = TRUE)
+summary(pca3)
+plot(pca3, type = "lines")
+loadings2 <- abs(pca3$rotation)
+loadings3 <- data.frame(prop.table(x = loadings2, margin = 2))
+loadings3[,1:4]
+# According to the loadings,we can see some relationships,
+# Group 1(Leadership programs): Change.Leadership, Politics, Economics.and.Education, Leadership, Arts.Administration
+# Group 2(Social Science programs):Social.Studies, Reading, Communication.Media.and.Learning.Technologies, Measurement..Evaluation.and.Statistics.
+# Group 3:History, Philosophy,Education.Leadership,Teaching.English,Adult.Education,Deaf.and.Hard.of.Hearing. 
+# Group 3 does not make much sense.
+# Group 4(Psychology programs):Developmental.Psychology,Psychology, Clinical.Psychology
```

Comment:
Very nice observation and interpretation of the PCs. 

> #Calculate values for each student that represent these your composite variables and then create a new correlogram showing their relationship to mean_correct.
Did you forget to calculate this? 

Good job! Keep up the good work in the future.